### ISSUE TITLE
Research and possibly remediate happy path failures around form.serviceInformation.TitlePastActiveDutyDate

### ISSUE THREAD

**Issue by austinjprice** [department-of-veterans-affairs/va.gov-team#69013]
Research and possibly remediate happy path failure 
- form.serviceInformation.TitlePastActiveDutyDate

## Acceptance Criteria
- [ ] Determine root cause of errors
- One of the following
  - [ ] New ticket created to fix issue
  - [ ] PR Opened to Fix issue
  - [ ] Comments on this ticket describing why it is not fixable at this time

[Datadog Link](https://vagov.ddog-gov.com/logs?query=%40named_tags.class%3A%22EVSS%3A%3ADisabilityCompensationForm%3A%3ASubmitForm526AllClaim%22%20%40level%3Aerror%20%40payload.submission_id%3A%282429126%20OR%202429129%20OR%202429132%20OR%202518075%29%20&cols=host%2Cservice%2C%40payload.error_message%2C%40payload.submission_id&context_event=AY2JIxEsAAB1aZOJsTXH6QAn&index=%2A&messageDisplay=inline&refresh_mode=paused&storage=online_archives&stream_sort=time%2Cdesc&viz=&from_ts=1698811200000&to_ts=1707414600000&live=false)

**Comment by freeheeling** [comment]
**DD query:** 
```
@named_tags.class:"EVSS::DisabilityCompensationForm::SubmitForm526AllClaim" @level:error @payload.error_message:"{\"messages\"=>[{\"key\"=>\"form526.serviceInformation.Title10PastActiveDutyDate\", \"severity\"=>\"ERROR\", \"text\"=>\"The Title 10 Activation Date for a claimant currently activated in the Reserves or National Guard is a future date\"}]}" 
```

152 instances over last year, with the following frequency distribution:
<img width="1707" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBODVuQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--873fd94ada2e9aaa4bee582aaa58230fc007c753/Screenshot%202024-04-16%20at%202.50.48%E2%80%AFPM.png" alt="Screenshot 2024-04-16 at 2.50.48 PM.png" />

Most recent occurring March 25.

**Comment by sortizsh** [comment]
@juliepedtke @nnicholas7  can you look at the copy Scott needs?

**Comment by freeheeling** [comment]
Labeling as `blocked` and moving to `Blocked` until copy is ready.

**Comment by juliepedtke** [comment]
@freeheeling letting you know we landed on this copy for the validation: "Your activation date must be in the past". We'd like to update the existing validation to match this format and remove the "please" - so "Your separation date must be less than 180 days in the future"

We'll note in our ticket as well, but we're moving away from using "please" throughout the form, so there are several other validation messages that will eventually need updating, but low priority. We also found a similar issue on the RNG page where there is no validation for the start date, and future dates can be entered (which shouldn't be possible for current Reservists).

This should unblock you. Let me know if I missed anything!

cc @nnicholas7

**Comment by freeheeling** [comment]
@juliepedtke – Do you know if the present date is considered a valid activation date, or must be in the past? I can reach out to EVSS for clarification, if you don't know. 

Also, I want to confirm that I should remove "Please" from all the following instances of error messages related to separation date and/or other error messages that appear on the Federal Orders page:
1.  **'Please enter a future separation date'**
2.  **'Please enter a separation date less than 180 days in the future'**
3.  **'Please enter an expected separation date that is after your activation date'**
4.  **'Please provide a valid year'**
5.  **'Please enter a date'**

Thanks!

**Comment by juliepedtke** [comment]
I haven't found anything definitive but I feel comfortable assuming that if someone had been activated that same day, it'd be most accurate to say they were currently activated (so present date is allowed). 

That's correct, moving away from please in all cases! Although this is making me notice there are two ways of phrasing things without "please":

- Enter a future separation date
- Your separation date must be in the future

I think the guidance is to go with the most concise, so can we actually change the new validation slightly to read "Enter an activation date in the past" so everything is consistent?

**Comment by freeheeling** [comment]
@juliepedtke – 'Please enter a date' is a validation that falls under Platform's purview as it's used throughout VA.gov. I'm not sure my making update to that messaging would pass review without some authoritative decree, but the other 4, plus new, validations utilize the more concise language.

**Comment by juliepedtke** [comment]
Interesting. Probably fine to leave it then, once the Content team comes out with formalized feedback on "please" Platform might update that. Not sweating it in the mean time.

**Comment by freeheeling** [comment]
[Draft PR](https://github.com/department-of-veterans-affairs/vets-website/pull/29392) ready for review.

**Comment by freeheeling** [comment]
Deployed [29392](https://github.com/department-of-veterans-affairs/vets-website/pull/29392) to prod.

<img width="1544" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBejZhQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--10746b0e1f7a6df0bc4f08fd69e4a867151e30e2/Screenshot%202024-04-26%20at%201.34.51%E2%80%AFPM.png" alt="Screenshot 2024-04-26 at 1.34.51 PM.png" />
<img width="1544" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBejJhQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c56310ca4131e408bbe8d007fc6017ed827f4868/Screenshot%202024-04-26%20at%201.34.30%E2%80%AFPM.png" alt="Screenshot 2024-04-26 at 1.34.30 PM.png" />

Moving to Done.
